components/brave_wallet_ui/components/desktop/popup-modals/sell-asset-modal/sell-asset-modal.tsx

```diff
@@ -0,0 +1,255 @@
+// Copyright (c) 2023 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// You can obtain one at https://mozilla.org/MPL/2.0/.
+
+import * as React from 'react'
+import {
+  useSelector
+} from 'react-redux'
+
+// Types
+import { WalletState, BraveWallet } from '../../../../constants/types'
+
+// Utils
+import { getLocale } from '../../../../../common/locale'
+import { computeFiatAmount, computeFiatAmountToAssetValue } from '../../../../utils/pricing-utils'
+import { CurrencySymbols } from '../../../../utils/currency-symbols'
+import Amount from '../../../../utils/amount'
+
+// Hooks
+import { useOnClickOutside } from '../../../../common/hooks/useOnClickOutside'
+
+// Components
+import PopupModal from '../../../desktop/popup-modals'
+import { NavButton } from '../../../extension'
+import { withPlaceholderIcon } from '../../../shared'
+
+import {
+  StyledWrapper,
+  InputSection,
+  Row,
+  AssetIcon,
+  Text,
+  AmountInput,
+  PresetButton,
+  ErrorBox,
+  ErrorIcon
+} from './sell-asset-modal.style'
+
+import { VerticalSpacer } from '../../../shared/style'
+
+interface Props {
+  selectedAsset: BraveWallet.BlockchainToken
+  selectedAssetsNetwork: BraveWallet.NetworkInfo | undefined
+  sellAmount: string
+  showSellModal: boolean
+  sellAssetBalance: string
+  setSellAmount: (value: string) => void
+  openSellAssetLink: () => void
+  onClose: () => void
+}
+
+export const SellAssetModal = (props: Props) => {
+  const {
+    selectedAsset,
+    selectedAssetsNetwork,
+    sellAmount,
+    showSellModal,
+    sellAssetBalance,
+    setSellAmount,
+    openSellAssetLink,
+    onClose
+  } = props
+
+  // Redux
+  const defaultCurrencies = useSelector(({ wallet }: { wallet: WalletState }) => wallet.defaultCurrencies)
+  const spotPrices = useSelector(({ wallet }: { wallet: WalletState }) => wallet.transactionSpotPrices)
+
+  // Refs
+  const sellAssetModalRef = React.useRef<HTMLDivElement>(null)
+
+  // Memos
+  const AssetIconWithPlaceholder = React.useMemo(() => {
+    return withPlaceholderIcon(AssetIcon, { size: 'medium', marginLeft: 4, marginRight: 8 })
+  }, [])
+
+  const fiatBalance = React.useMemo(() => {
+    return computeFiatAmount(
+      spotPrices,
+      {
+        decimals: selectedAsset?.decimals ?? '',
+        symbol: selectedAsset?.symbol ?? '',
+        value: sellAssetBalance
+      })
+  }, [spotPrices, sellAssetBalance, selectedAsset?.symbol, selectedAsset?.decimals, defaultCurrencies.fiat])
```

Comment:
Remove `defaultCurrencies.fiat` from deps list. It is not used in the function

Reply:
Good catch! Removed!